### PR TITLE
Added DIGITAL_CREDENTIALS_SUPPORTED_RUNS configuration

### PR DESCRIPTION
**Pull request by arslanashraf7**
#### What are the relevant tickets?
https://github.com/mitodl/mitxpro/issues/2181

#### What's this PR do?
Adds configuration key `DIGITAL_CREDENTIALS_SUPPORTED_RUNS` and value in RC and PROD

#### How should this be manually tested?
- Make sure that this PR(https://github.com/mitodl/mitxpro/pull/2182) is deployed.
- Make sure that the `FEATURE_DIGITAL_CREDENTIALS` is turned on.
- Make sure that only specified courses/programs show digital credentials link on the dashboard.

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)

### CODE REVIEW

**Comment by shaidar** [comment]
This needs an entry in `ci` otherwise pillar rendering would fail.

**Comment by arslanashraf7** [comment]
> This needs an entry in `ci` otherwise pillar rendering would fail.

@shaidar, Thanks for pointing it out, I've added the entry in ci with empty value believing that would not create any trouble. Let me know if there could still be a problem.

Also, I'll merge it once approved by @rhysyngsun.

**Comment by shaidar** [comment]
👍

**Review by rhysyngsun — APPROVED** [review]
LGTM

**Pull request merged**
